tests/rules/assertions/requireReadonlyReactProps.js

```diff
@@ -194,5 +210,13 @@ export default {
     {
       code: 'function Foo(props: {||}) { return <p /> }',
     },
+    {
+      code: 'function Foo(props: {}) { return <p /> }',
+      options: [
+        {
+          implicitExactTypes: true,
+        },
+      ],
+    },
```

Comment:
I'm just thinking for an extra valid case, would you wanna use `{| |}` syntax and with `implicitExactTypes: true,`. Just to test that it doesn't blow up for an edge case?

Reply:
Sure, good idea. 👍